platform/README.md

```diff
@@ -0,0 +1,73 @@
+Guice Platform Bill of Materials (BOM)
+============================
+This BOM is an extension of of the Guice BOM. It appends all dependencies needed to run Guice and its extensions.
```

Comment:
The Guice BOM represents Guice Components. The Guice Platform represents the Guice Components and all those dependencies that make Guice work.